__tests__/app.test.js

```diff
@@ -39,3 +39,42 @@ describe("GET /api/categories", () => {
       });
   });
 });
+
+describe("GET /api/reviews/:review_id", () => {
+  it("404: when accessing a nonexistant review", () => {
+    return request(app).get("/api/reviews/14").expect(404);
+  });
+
+  it("400: when supplying an invalid review id (not a number)", () => {
+    return request(app)
+      .get("/api/reviews/aaaa")
+      .expect(400)
+      .then(({ body }) => {
+        const { msg } = body;
+        expect(msg).toBe("Bad Request");
+      });
+  });
+
+  it("200: returns a review object with correct properties", () => {
+    const path = "/api/reviews/";
+    for (let index = 1; index < 14; index++) {
+      return request(app)
+        .get(path + String(index))
+        .expect(200)
+        .then(({ body }) => {
+          const { review } = body;
+          expect(review).toMatchObject({
+            review_id: expect.any(Number),
+            title: expect.any(String),
+            review_body: expect.any(String),
+            designer: expect.any(String),
+            owner: expect.any(String),
+            review_img_url: expect.any(String),
+            votes: expect.any(Number),
+            category: expect.any(String),
+            created_at: expect.any(String),
+          });
+        });
```

Comment:
Like your thinking here! We're making sure it works for every item in the database.
However there's a couple of things to consider:
- We are running 14 GET requests every time this test runs, that's quite a lot of processing power when we potentially only need to do one implicit check to prove this endpoint works.
- We aren't proving that requesting for example id 1, gives us the exact response we expect with the exact data we know it should have.

Consider refactoring to assert one request with one id gives us an exact match rather than expect.any.